static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -50,4 +52,194 @@
+        border: solid 1px $neutral-5;
+        margin: $gs-gutter/4 0 $gs-baseline*2;
+        padding: $gs-baseline/2 $gs-gutter/2 $gs-baseline;
+        position: relative;
+        clear: none;
+        float: none;
+        width: inherit;
+        @include mq(mobile) {
+            margin-left: -$gs-gutter/2 - 1px;
+        }
+        @include mq(mobileLandscape) {
+            margin-right: $gs-gutter/2;
+            margin-left: -$gs-gutter;
+        }
+        @include mq(leftCol) {
+            margin-left: -$gs-baseline*2;
+        }
+        // Quote tail
+        &:after {
+            @include mq(mobile) {
+                content: '';
+                width: $quote-tail;
+                height: $quote-tail;
+                border-radius: 0 0 $quote-tail;
+                position: absolute;
+                bottom: -$quote-tail - 2px;
+                left: $gs-gutter/2;
+                background-color: $neutral-garnett-1;
+                border: solid 1px $neutral-5;
+                border-top: solid 1px $neutral-garnett-1;
+            }
+            @include mq(mobileLandscape) {
+                left: $gs-gutter;
+            }
+            @include mq(desktop) {
+                left: $gs-gutter - $gs-baseline;
+            }
+            @include mq(leftCol) {
+                left: $gs-baseline;
+            }
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            display: inline;
+        }
+
+        .inline-quote,
+        blockquote {
+            display: inline;
+        }
+        // Hack hide for svg
+        .closing {
+            display: none;
+        }
+    }
+    // Quote supporting
+    .element-pullquote.element--supporting {
+        @include mq(mobileMedium) {
+            clear: left;
+            float: left;
+            margin-right: $gs-gutter/2;
+            width: gs-span(3);
+        }
+        @include mq(leftCol) {
+            margin-left: - gs-span(3)/2 - $gs-gutter/2;
+        }
+
+        &:after {
+            @include mq(leftCol) {
+                border-radius: 0 0 0 $quote-tail;
+                left: gs-span(3)/2 - 28px;
+            }
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            @include fs-headlineGarnett(1);
+        }
+    }
+    // Quote halfWidth
+    .element-pullquote.element--halfWidth {
+        @include mq(mobileMedium) {
+            clear: left;
+            float: left;
+            margin-right: $gs-gutter/2;
+            width: gs-span(3);
+        }
+        @include mq(mobileLandscape) {
+            width: gs-span(4);
+        }
+
+        .pullquote-cite,
+        .pullquote-paragraph {
+            @include mq(mobile) {
+                @include fs-headlineGarnett(2);
+            }
+            @include mq(desktop) {
+                @include fs-headlineGarnett(3);
+            }
+        }
+    }
+    // Quote inline
+    .element-pullquote.element--inline {
```

Comment:
as above. woudl this be captured with `element-pullquote--inline`?

Reply:
also, this may exist already, so np if so, but does the name `element-pullquote--inline` capture anything that `pullquote--inline` woudl not?

Reply:
Yea, I've pulled in these class names from the existing markup (including the camel case). As far as I can tell the only way of distinguishing them from other inline elements is by concatenating them to 'element-pullquote', is there a better solution? 

<img width="546" alt="screen shot 2017-12-01 at 16 34 34" src="https://user-images.githubusercontent.com/8453924/33492640-92ef975c-d6b5-11e7-98bd-bd4743db611f.png">

Reply:
no taht's cool - they're 'wrong' but if they're present already lets not worry about it